ui/app/src/components/FormsEngine/controls/VideoPicker.tsx

```
@@ -0,0 +1,256 @@
+		});
+
+		const menuOptions = [];
+		// TODO: pending when datasources implementation is ready.
+		const handleDataSourceOptionClick = (event: ReactMouseEvent<HTMLLIElement, MouseEvent>, option: PickerType) => {};
+		if (allowedBrowsePaths.length > 0) {
+			menuOptions.push(
+				<MenuItem key="search" onClick={(event) => handleDataSourceOptionClick(event, 'browse')}>
+					<ListItemIcon sx={{ mr: 0 }}>
+						<TravelExploreOutlined fontSize="small" />
+					</ListItemIcon>
+					<ListItemText children={<FormattedMessage defaultMessage="Browse" />} />
```

Biome (2.4.4)

[error] Avoid passing children using a prop

(lint/correctness/noChildrenProp)